libibverbs/driver.h

```diff
@@ -420,8 +420,7 @@ struct verbs_context_ops {
 			  struct ibv_port_attr *port_attr);
 	int (*query_qp)(struct ibv_qp *qp, struct ibv_qp_attr *attr,
 			int attr_mask, struct ibv_qp_init_attr *init_attr);
-	int (*query_qp_data_in_order)(struct ibv_qp *qp, enum ibv_wr_opcode op,
-				      uint32_t flags);
```

Comment:
I would keep the flags here, the provider can choose to ignore them.

Reply:
The idea of this interface change is that providers will now return all their supported capabilities (related to data inorder) without depending on the request. I think flags here may be confusing, when would you use it in provider code?

Reply:
I don't know, depends on the flags.
Anyway, it's not a big deal, I guess the flags can be reintroduced if/when they're needed.